# HG changeset patch
# User mintsuki
# Date 1576755327 -3600
#      Thu Dec 19 12:35:27 2019 +0100
# Node ID d6b591ffb37f8aac83c49f3ec7e6d56db311e109
# Parent  39a28a4033f5e22b6d3d98ca18c520443b326345
Work around wayland-scanner issue.

diff -r 39a28a4033f5 -r d6b591ffb37f cmake/sdlchecks.cmake
--- a/cmake/sdlchecks.cmake	Sat Nov 30 01:13:06 2019 +0100
+++ b/cmake/sdlchecks.cmake	Thu Dec 19 12:35:27 2019 +0100
@@ -597,7 +597,7 @@
 
     if(WAYLAND_FOUND)
       execute_process(
-        COMMAND ${PKG_CONFIG_EXECUTABLE} --variable=wayland_scanner wayland-scanner
+        COMMAND wayland-scanner
         WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
         RESULT_VARIABLE WAYLAND_SCANNER_RC
         OUTPUT_VARIABLE WAYLAND_SCANNER
diff -r 39a28a4033f5 -r d6b591ffb37f configure.ac
--- a/configure.ac	Sat Nov 30 01:13:06 2019 +0100
+++ b/configure.ac	Thu Dec 19 12:35:27 2019 +0100
@@ -1453,7 +1453,7 @@
             if $PKG_CONFIG --exists wayland-client wayland-scanner wayland-protocols wayland-egl wayland-cursor egl xkbcommon ; then
                 WAYLAND_CFLAGS=`$PKG_CONFIG --cflags wayland-client wayland-egl wayland-cursor xkbcommon`
                 WAYLAND_LIBS=`$PKG_CONFIG --libs wayland-client wayland-egl wayland-cursor xkbcommon`
-                WAYLAND_SCANNER=`$PKG_CONFIG --variable=wayland_scanner wayland-scanner`
+                WAYLAND_SCANNER=wayland-scanner
                 video_wayland=yes
             fi
         fi
